### PR TITLE
tox: add page

### PR DESCRIPTION
**Pull request by tdhopper**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/tox.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# tox
+
+> Automate Python testing across multiple Python versions.
+> Use tox.ini to configure environments and test command.
+
+- Run tests on all test environments:
+
+`tox`
+
+- Create tox.ini configuration:
```
Comment (sbrl) [inline pages/common/tox.md]: Perhaps `Create a tox.ini configuration` would read better?

File: pages/common/tox.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`tox-quickstart`
+
+- List available environments:
```
Comment (sbrl) [inline pages/common/tox.md]: Perhaps `List the available environments` would read better?

File: pages/common/tox.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`tox --listenvs-all`
+
+- Run tests on a specific environment (e.g. python 3.6):
+
+`tox -e py36`
```
Comment (sbrl) [inline pages/common/tox.md]: Is there a long form of the `-e` argument here? Here at tldr-pages we prefer long-form options in pages, as they make them more straight-forward for new users. For example, `ls -a` is short for `ls --all`.

File: pages/common/tox.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Force recreation of virtual environment:
```
Comment (sbrl) [inline pages/common/tox.md]: Perhaps `Force the virtual environment to be recreated` would read better?

Also, same here with the `-e`.

File: pages/common/tox.md (hunk 3 above)
Comment (agnivade) [reply pages/common/tox.md]: Also, py36 in tokens.

File: pages/common/tox.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`tox --recreate -e py27`
```
Comment (agnivade) [inline pages/common/tox.md]: py27 in tokens (`{{}}`) please.

File: pages/common/tox.md (hunk 3 above)
Comment (tdhopper) [reply pages/common/tox.md]: `tox --help` shows

```
  -e envlist            work against specified environments (ALL selects all).
```

Comment (sbrl) [reply pages/common/tox.md]: Ah, ok. No worries :-)

**Comment by sbrl** [comment]
Thanks, @tdhopper :smiley_cat:

**Pull request merged**